src/main/java/org/opentripplanner/standalone/GrizzlyServer.java

```diff
@@ -131,7 +136,8 @@ public void run() {
         } catch (InterruptedException ie) {
             LOG.info("Interrupted, shutting down.");
         }
-        httpServer.shutdown();
```

Comment:
Good question. Initially I kept the hook, but this method is potentially called multiple times from `OTPMain`. However, I now see that I should not have removed the call to `httpServer.shutdown()`. Adding a commit that puts it back.